### PR TITLE
Publish release notes for Godot 4.1 RC 1

### PR DESCRIPTION
**Pull request by YuriSizov**
This only contains the blogpost. I've pushed the new version to the list earlier today.

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-1-rc-1.webp
+image_caption_title: "Katana Dragon"
+image_caption_description: "A game by Tsunoa Games"
+date: 2023-06-27 15:00:00
+---
+
+Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Since the release of beta 3 we have identified all remaining regressions, thanks in a large part to your reports and extensive testing, and with the necessary fixes implemented we are publishing the first [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) today.
+
+This is a great opportunity to give the new version of the engine a try. As the name suggests, a release candidate represents a build that we consider stable but want to be absolutely sure before making it official, — with your help. If there are no significant regressions reported with release candidates, a stable version of Godot 4.1 is going to be published soon.
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md]:
```suggestion
This is a great opportunity to give the new version of the engine a try. As the name suggests, a release candidate represents a build that we consider stable but want to be absolutely sure before making it official — with your help. If there are no significant regressions reported with release candidates, a stable version of Godot 4.1 is going to be published soon.
```

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+-----
+
+*The illustration picture for this article is from* [**Katana Dragon**](https://twitter.com/KatanaDragon_) *— an action RPG developed by [Tsunoa Games](https://twitter.com/tsunoagames) with Godot 4. When the game releases you will be able to venture into a charming voxel world to explore dungeons, avoid traps, and solve puzzles! Follow the development of the game on [Twitter](https://twitter.com/KatanaDragon_) so you don't miss the release date.*
+
+## What's new
+
+41 contributor submitted around 70 improvements for this release. You can review the complete list of changes with our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.1-rc1), which contains links to relevant commits and PRs for this and every previous release.
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md]:
```suggestion
41 contributors submitted around 70 improvements for this release. You can review the complete list of changes with our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.1-rc1), which contains links to relevant commits and PRs for this and every previous release.
```

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+title: "Release candidate: Godot 4.1 RC 1"
+excerpt: "After 4 months of development, Godot 4.1 is only days away. To make sure it doesn't miss the mark, here's the first release candidate for your testing!"
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-1-rc-1.webp
+image_caption_title: "Katana Dragon"
+image_caption_description: "A game by Tsunoa Games"
+date: 2023-06-27 15:00:00
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md]: Let's go live now :)
```suggestion
date: 2023-06-27 14:00:00
```

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+- Core: Ensure `RID`, `Callable`, and `Signal` are stored as strings ([GH-78517](https://github.com/godotengine/godot/pull/78517)).
+- Core: Fix scene load crash related to `_ready` ([GH-78654](https://github.com/godotengine/godot/pull/78654)).
+- Core: Ensure default node groups' call queue are processed ([GH-78713](https://github.com/godotengine/godot/pull/78713)).
+- Editor: Handle contextual editors gracefully when restoring layout ([GH-78611](https://github.com/godotengine/godot/pull/78611)).
+- Editor: Display a message about missing C# support on Android/iOS/Web platforms ([GH-78629](https://github.com/godotengine/godot/pull/78629)).
+- Editor: Improve script icon cache ([GH-78670](https://github.com/godotengine/godot/pull/78670)).
+- GDExtension: Fix text_server_adv compiling as a GDExtension ([GH-77532](https://github.com/godotengine/godot/pull/77532)).
+- GDExtension: Add GDExtension `@since` attribute ([GH-78518](https://github.com/godotengine/godot/pull/78518)).
+- GDScript: Fix a race condition in ScriptServer ([GH-76586](https://github.com/godotengine/godot/pull/76586)).
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md:43]: Technically more of a core change, I relabeled the issue. But it doesn't matter much, can be left as is here.

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- Rendering: Reset filter/repeat state of textures in GL Compatibility when render target is cleared ([GH-78620](https://github.com/godotengine/godot/pull/78620)).
+- Rendering: Use a filter with mipmaps when initializing textures with mipmaps in GL Compatibility renderer ([GH-78720](https://github.com/godotengine/godot/pull/78720)).
+- XR: Fix OpenXR Passthrough mode ([GH-78135](https://github.com/godotengine/godot/pull/78135)).
+- XR: Apply reprojection in multiview for our cluster lookup ([GH-78499](https://github.com/godotengine/godot/pull/78499)).
+- XR: Fix incorrect depth buffer option in OpenXR ([GH-78550](https://github.com/godotengine/godot/pull/78550)).
+- Documentation and translation updates.
+
+A previously included fix has been reverted due to regressions and will be addressed in a future version of Godot:
+
+- Editor: Revert "Fix paste value not updated in dictionaries/arrays" ([GH-78643](https://github.com/godotengine/godot/pull/78643)).
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md:65]: Nice!

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+This release is built from commit [`1f9e540f1`](https://github.com/godotengine/godot/commit/1f9e540f14edbf2d496a1421f8d37e5b483c4c53) (see [README](https://downloads.tuxfamily.org/godotengine/4.1/rc1/README.txt)).
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md:67]: :gdunacceptable: style change! Just kidding, it makes sense :)

File: collections/_article/release-candidate-godot-4-1-rc-1.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+* [Standard build](https://downloads.tuxfamily.org/godotengine/4.1/rc1/) (GDScript, GDExtension).
+* [.NET 6 build](https://downloads.tuxfamily.org/godotengine/4.1/rc1/mono) (C#, GDScript, GDExtension).
+  - Requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) installed in a standard location.
+
+## Known issues
+
+There are currently no known issues introduced by this release.
+
+With every release we accept that there are going to be various issues, which have already been reported but haven't been fixed yet. See the GitHub issue tracker for a list of [known bugs in the 4.1 milestone](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+milestone%3A4.1+label%3Abug+).
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-rc-1.md]:
```suggestion
With every release we accept that there are going to be various issues, which have already been reported but haven't been fixed yet. See the GitHub issue tracker for a list of [known bugs in the 4.1 milestone](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+milestone%3A4.1+label%3Abug+).

Some notable issues are still tracked for a potential fix before 4.1-stable:
- The processing order for internal and normal node processing was mistakenly changed, causing issues such as [GH-77548](https://github.com/godotengine/godot/issues/77548). You can help us test [GH-78745](https://github.com/godotengine/godot/pull/78745) to validate that it solves the issue without further regression.
- The editor output dock doesn't update its scrollbar properly in some cases ([GH-78434](https://github.com/godotengine/godot/issues/78434)). A potential solution is identified but we're not sure yet that it's safe to include in 4.1 at this late stage given the risk of regression.
```

**Review by akien-mga — APPROVED** [review]

File: collections/_article/release-candidate-godot-4-1-rc-1.md (hunk 6 above)
Comment (YuriSizov) [reply collections/_article/release-candidate-godot-4-1-rc-1.md:67]: I copied it from RCs for something else 🤔

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**